### DISCUSSION TITLE
Separate build of master, filler and server volume

### DISCUSSION THREAD

**Discussion by kmlebedev** [seaweedfs/seaweedfs#3019] [General]
Hi, There is an idea to be able to make independent builds of Master, filler and server volume.
Why is this needed? I want to decompose the task of porting seaweedfs to different architecture.
But I have doubts, it seems binary protobuff format metadata platform dependent

**Comment by chrislusf** [comment]
the protobuf and metadata are platform independent.

You can always safely mix them as long as the version is the same.
